### ISSUE TITLE
listrunner doesn't work reliably in agent mode

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#198]
Originally reported of Google Code with ID 142.

```
When used in agent mode, it doesn't interact well with the ssh-agent, as all children reuse the same connection and this leads to breakage.

Each child should reopen a connection to the agent, or another method should be used to ensure reliable operation.
```
Originally added on 2011-02-16 11:45:41 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
This has been fixed and will be in the final 2.4 release.
```
Originally added on 2011-02-16 16:46:10 +0000 UTC.

Changed State: Fixed
Added to Milestone: Release2.4

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 08:57:08 +0000 UTC.

Changed State: Released
